gandalf.js

```diff
@@ -10,27 +10,25 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?